pages/common/qr.md

```diff
@@ -0,0 +1,12 @@
+# qr
+
+> Generate QR codes in the terminal with ANSI VT-100 escape codes.
+> More information: <https://github.com/lincolnloop/python-qrcode/>.
+
+- Generate a QR code:
+
+`echo "{{data}}" | qr`
+
+- Specify the error correction level (defaults to M):
```

Comment:
Maybe it'd be better to explain the qualitative meaning of each level, i.e. what does each of L, M, Q and H mean.